typeDefs.js

```diff
@@ -365,6 +367,7 @@ const typeDefs = gql`
     createNovaPoshtaOrder(data: NovaPoshtaOrderInput): NovaPoshtaOrderResult
     getUkrPoshtaRegion(region: String): UkrPoshtaRegion
     getPaymentCheckout(data: PaymentInput): Payment
+    getPaymentStatus(orderId: String): PaymentStatus
```

Comment:
It's make sense to set this field as required.

Reply:
fixed